model_analyzer/config/generate/generator_utils.py

```diff
@@ -90,3 +91,10 @@ def generate_parameter_combinations(params):
 
         param_combinations = list(product(*tuple(params.values())))
         return [dict(zip(params.keys(), vals)) for vals in param_combinations]
+
+    def generate_log_list(value):
+        """
+        Returns a list of exponential numbers from 1 to value
+        """
+        log_value = int(log2(value))
+        return [2**c for c in range(0, log_value + 1)]
```

Comment:
endline at end of all files. But -- nice new function! I wouldn't have thought to add this instead of just putting the functionality in PACG 👍